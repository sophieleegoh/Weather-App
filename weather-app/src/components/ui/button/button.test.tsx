import renderer from "react-test-renderer";
import Button from "./index";

describe("Button snapshot tests", () => {
  it("Renders the component", () => {
    const component = renderer.create(
      <Button buttonLabel="button label" onUpdate={() => {}} />
    );

    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
