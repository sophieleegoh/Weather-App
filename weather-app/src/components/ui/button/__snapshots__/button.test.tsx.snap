// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snapshot tests Renders the component 1`] = `
<div
  className="py-4"
>
  <button
    className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline"
    onClick={[Function]}
  >
    button label
  </button>
</div>
`;
